src/views/dog/MinigameJumpNRun.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div class="game-container" :style="{ backgroundPositionX: `${backgroundPositionX}px` }">
+    <Character :isJumping="isJumping" />
+    <Obstacle :image="StoneImg" :positionX="obstaclePositionX" />
+    <BtnControl type="jump" @jump="jump" bottom="10" left="100" />
+    <button @click="goToNextStage">Nächstes Minigame</button>
+    <Goal v-if="isGoalVisible" :positionX="goalPositionX" />
+
+    <PooComponent
+      v-for="poo in poos"
+      :key="poo.id"
+      :id="poo.id"
+      :positionX="poo.positionX"
+      @collect="collectPoo(poo.id)"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import Character from '@/components/JumpNRun/JumpNRunCharacter.vue'
+import Obstacle from '@/components/JumpNRun/ObstacleItem.vue'
+import PooComponent from '@/components/JumpNRun/PooComponent.vue'
+import BtnControl from '@/components/JumpNRun/ControlButton.vue'
+import { useStageNavigator } from '@/composables/useNavigation'
+import { useMascotStore } from '@/stores/useMascotStore'
+import mascotMessages from '@/config/mascotMessages'
+import StoneImg from '@/assets/jumpNrun/stone.png'
+import Goal from '@/components/JumpNRun/GoalComponent.vue'
+
+const { goToNextStage } = useStageNavigator()
+const mascot = useMascotStore()
+const jumpNRunMessages = mascotMessages.dog.stage3
+
+const initialBackgroundPositionX = 0
+const initialObstaclePositionX = 2000
+const initialIsRunning = false
+const initialIsJumping = false
+const initialIsWaiting = false
+
+let backgroundPositionX = ref(initialBackgroundPositionX)
```

Comment:
All the variables should be constatnts (replace let with const)

Reply:
Or you also can check if it makes more sense to use a single reactive with all attributes in it, instead of single ref's